index.html

```diff
@@ -0,0 +1,158 @@
+  <title>LeMorsels</title>
+  <link rel="stylesheet" href="CSS/styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  
+  <div class="b-background-i">
+  
+  <header class="toplines header">
+    <div class="top-menu">
+      <div class="top-menu-container">
+        <div class="top-menu-actions">
+          <a href="#" class="mini-facebook"><img src="images/white-f.png" alt="Facebook"></a>
+             <a href="#" class="mini-twitter"><img src="images/twitter-w.png" alt="twitter"></a>
+                <h5><a>English</a></h5>
+                <h5><a>My Page</a></h5>
+                <h5><a>Logout</a></h5>
+             </div>
+         </div>
+     </div>
+
+     <nav class="menus-bar">
+
+       <!-- mobile menu dynamic -->
+
+      <div class="menu-icon"><img src="./images/toggle-menu.png" id="KrabbyPatty" alt="burger-menu"/></div>
+        
+      <div class="mobile-minus isHidden">
+        <img class="close-btn" src="./images/x-mark.png"/>
+        <ul class="menu-list">
+           <li><a href="./abouts.html" class="link-nav">About</a></li>
+           <li><a href="#" class="link-nav">Program</a></li>
+           <li><a href="#" class="link-nav">Sponsor</a></li>
+           <li><a href="#" class="link-nav">Join</a></li>
+        </ul>
+      </div>
+
+       <!-- end of mobile menu dynamic -->
+
+    <div class="desktop-menus">
+        <a href="#" class="le-logo"><img src="./images/lemorsels-logo.PNG" alt="#" class="le-logo2"></a> 
+
+         <div class="desktop-nav-lnks">
+            <ul class="menu-ul-lnks">
+                <li><a href="./abouts.html" class="menu-links">About</a></li>
+                <li><a href="#" class="menu-links">Program</a></li>
+                <li><a href="#" class="menu-links">Join</a></li>
+                <li><a href="#" class="menu-links">Sponsor</a></li>
+                <li><a href="#" class="menu-links">News</a></li>
+            </ul>
+            <button class="menu-button" type="button">AS Campaign</button>
+          </div>
+        </div>
+        
+      </nav>
+  </header>
+      
+         <section class="headline-container">
+
+            <div class="Le-title">
+               <p class="catch-phrase">"Hello, organic food lovers!"</p>
+               <h1 class="Le-Slogan">LeMorsels, IT'S NATURAL AND ORGANIC AS IT SHOULD BE!</h1>
+            </div>
+          <div class="below-slogan">
+            <p class="Lemorse-quote">
+            Healthy eating is a way of life, so it's important to establish routines that are simple, realistically, and ultimately livable. 
+            LeMorsels is working with over 80 countries to help their communites to gain a better undertanding of a healthy meal life.</p>
+            <div class="small-info">
+              <p class="lemorse-date">2022.09.12(MON) ~ 16(FRI)</p>
```

Comment:
- I think it's better to use another font for numbers. Your project must present useful information instead of unreadable signs.